README.md

```diff
@@ -77,6 +77,12 @@ Results are stored in `demo-dir` by default
 - `defaultWaitUntil`
   - default: 'networkidle2'
   - [Other options](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagegotourl-options)
+- `executablePath`
+  - Path to Chromium executable.
+  - default: uses bundled puppeteer chromium
+- `extraChromiumArgs`
```

Comment:
I tried passing `["--accept-lang=es-ES"]` to this and it didn't work (it does work independently of the collector), I wonder if there's some other arg I could try?

Reply:
This works if you are in headful mode, showing it's passing the arg correctly. For some reason chrome doesn't send that header in headless mode.

Reply:
Not our bug!